mkdocs.yml

```diff
@@ -25,7 +25,7 @@ nav:
     - 'Extensions':
         - 'RxJava': android_sqlite/rxjava.md
         - 'Coroutines': android_sqlite/coroutines.md
-        - 'Android Paging': android_sqlite/android_paging.md
+        - 'AndroidX Paging': android_sqlite/androidx_paging.md
```

Comment:
Do you want to also add this to the `Mutliplatform` section?

Reply:
A very late follow up PR https://github.com/cashapp/sqldelight/pull/4265